.travis.yml

```diff
@@ -1,9 +1,11 @@
 language: python
+dist: xenial
 python:
-  - "2.7"
+  - "3.7"
```

Comment:
i think you need both if you want to run some tests on 2.7 and some tests on 3.7

Reply:
Negative. Having both just ran all the tests twice. And the python3 tests via python2 don't work at all.

Reply:
I tried that, it didn't work.